src/js/misc.js

```diff
@@ -21,6 +21,8 @@ $('#link-about').click(function(e) {
   fileInput.clear();
   // reset counts
   count.reset();
+  $('#fileError').addClass('hide');
+  $('#errorMessage').addClass('hide');
   // clear tables
```

Comment:
It might make sense to move these three into their own `reset` function, as it seems like something that could be more broadly reused.